verdigado_attendance/security/verdigado_attendance.xml

```diff
@@ -167,6 +167,6 @@
 
     <record id="base.default_user" model="res.users">
         <!-- don't make new users hr managers by default /-->
-        <field name="groups_id" eval="[(5, ref('hr.group_hr_manager'))]" />
+        <field name="groups_id" eval="[(3, ref('hr.group_hr_manager'))]" />
```

Comment:
das war der folgenschwere Tippfehler (statt nur den Manager zu löschen leert 5 das ganze Feld), der Rest ist aufräumen.

Es kann sein dass dieser Bug bisher geholfen hat nicht Benutzer mit zuvielen Rechten zu erstellen - deshalb schlage ich vor die Liste der Default-Gruppen die ich oben wiederherstelle zu kontrollieren, und dort rauszunehmen was nicht alle neuen Benutzer haben sollen.

Falls Du oben etwas entfernst sollte das dann hier auch mit einer `(3, ref('xmlid'))` eingetragen werden um dafür zu sorgen dass bei neuen Datenbanken bei dem dieses Modul installiert wird die default-Gruppen auch angepasst werden.

Reply:
Ganz verstehe ich es nicht. 

Vom Verhalten her funktioniert es jetzt, dass
* Administrator kann neue Benutzer anlegen
* Developer Mode ist nicht notwendig

Was mich irritiert: Die Standard-Rechte für neue Benutzer sind jetzt ne Mischung aus "Personalverantwortlicher" und "Manager". Das sollte so nicht bleiben.

Initial wäre wünschenswert, dass
* entweder alle Felder / Rechte unter Human Resources leer / nicht gesetzt sind
* oder: dass die Option Other -> Verdigado auf "Mitarbeiter" gesetzt ist und die Rechte "Humand Resources" entsprechend gesetzt sind.

![2024-01-11_10-43](https://github.com/verdigado/odoo-customize/assets/3354500/181af992-b713-4def-97bf-5b27aa8b2231)

Reply:
ja, das sind die xmlids der Gruppen die ich im [Migrationsscript](https://github.com/verdigado/odoo-customize/pull/94/files#diff-12939ecbe6d5bb54c55d6064c59af7350d53573773529e3d132d86faaccf175dR8-R21) setze, und das ist was Standard-Odoo macht (das kannst Du vergleichen durch diesen Codestand in einer leeren Datenbank zu installieren).

Also oben die `_manager`-Gruppen wegnehmen und `verdigado_attendance.group_verdigado_employee` zufügen?

Reply:
Ja, ich denke, das sollte der Weg sein.

Das wundert mich wirklich, dass `hr_holidays` und `hr_attendance` offenbar "Administrator" Rechte für HR vergeben, wenn man Benutzer anlegt. Das ist doch eher unüblich, oder?

Reply:
ja, hr ist hier der Ausreißer, normalerweise gehören hier die _user Gruppen rein. Folgt dem Mantra dass alles 'easy' sein soll...

Migrationsscript und default user sind jetzt angepasst, das sollte also dasselbe Ergebnis bei Neuinstallation sowie beim Upgrade auf Version 1.0.2 liefern.

Reply:
Jetzt verhält es sich so, dass bei neuen Usern `Other -> Verdigado` mit "Mitarbeiter*in" gesetzt ist. --> super.

Aber die HR-Rechte sind auf "Personalverantwortlicher" gesetzt. Mach ich was falsch?

![2024-01-15_09-41](https://github.com/verdigado/odoo-customize/assets/3354500/df71b438-8db4-4d29-8ed9-3741a1fee8cb)

Reply:
nein, das sind auch default-Gruppen die core setzt. Also auch `hr_attendance.group_hr_attendance_user`, `hr_holidays.group_hr_holidays_user` und `hr.group_hr_user` von Standarduser wegnehmen?

Reply:
Ich denke schon. Das Ziel sollte sein, dass es wie folgt aussieht, wenn man neue Benutzer anlegt:

Human Resources:
- Personal: leer
- Abwesenheiten: leer
- Anwesenheiten: Manuelle Anwesenheit

Other -> Verdigado: Mitarbeiter*in

Reply:
das passiert jetzt